linetools/spectra/tests/test_xspectrum_utils.py

```diff
@@ -149,6 +149,8 @@ def test_copy(spec):
     assert spec.wavelength[0] == spec2.wavelength[0]
     assert spec.flux[-1] == spec2.flux[-1]
 
+def test_plot(spec):
```

Comment:
I've thought about this.  For the code too.
I think it needs a bit more thought first or a 
commitment to a full Refactor.  Punting for now.